test/suites/vcf/mtdna_blb.sh

```diff
@@ -13,7 +13,7 @@ args+=("--resume")
 runVip "${args}" "${TEST_RESOURCES_DIR}/mtdna_blb.tsv"
 
 # compare expected to actual output and store result
-if [ "$(zcat "${OUTPUT_DIR}/vip.vcf.gz" | grep -vc "^#")" -le 85 ]; then
+if [ "$(zcat "${OUTPUT_DIR}/vip.vcf.gz" | grep -vc "^#")" -le 143 ]; then
```

Comment:
Just wondering: do you know which of the scores is the biggest contributor to this change, Apagee or Mitotip